go/stellar/transform.go

```diff
@@ -200,16 +222,19 @@ func transformPaymentRelay(mctx libkb.MetaContext, acctID stellar1.AccountID, p
 		}
 		loc.ToUsername = username
 		loc.ToType = stellar1.ParticipantType_KEYBASE
+		toName = username
 	}
 
 	if p.TxStatus != stellar1.TransactionStatus_SUCCESS {
-		// If the funding tx is not complete
+		// The funding tx is not complete.
 		loc.StatusSimplified = p.TxStatus.ToPaymentStatus()
 		loc.StatusDetail = p.TxErrMsg
 	} else {
 		loc.StatusSimplified = stellar1.PaymentStatus_CLAIMABLE
-		loc.StatusDetail = "Waiting for the recipient to open the app to claim, or the sender to cancel."
-		loc.ShowCancel = true
+		if isSender {
+			loc.StatusDetail = fmt.Sprintf("%v can claim this when they set up their wallet.", toName)
```

Comment:
This "can claim" value is StatusDetail is not used. Because it can get overridden by an error message from a failed claim. If `PaymentLocal` had another string field for this then the frontend could lean on the service for this message. But I don't want to make `PaymentLocal` any bigger unless someone thinks it's worth it.